### PR TITLE
Removes absence of inclusive pins

### PR DESCRIPTION
**Pull request by intercepti0n**
I was very disappointed when i found out absence of pride pins 'cause players can't express their diversity in game. New pride pins add inclusive role-play options to Aurora. I think that it's a good thing!

Source: [ChaoticOnyx/OnyxBay#11280](https://github.com/ChaoticOnyx/OnyxBay/pull/11280)

Wow, amazing!

### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| icons/clothing/accessories/pins.dmi | [TobyThorne](https://github.com/TobyThorne) (OnyxBay) | AGPL-3.0 |
| icons/mob/pins.dmi | [TobyThorne](https://github.com/TobyThorne) (OnyxBay) | AGPL-3.0 |

### CODE REVIEW

**Comment by intercepti0n** [comment]
!review

**Review by kreeperHLC — APPROVED** [review]

File: code/modules/clothing/under/accessories/pins.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+/obj/item/clothing/accessory/pridepin/pan
+	name = "pansexual pride pin"
+	icon_state = "pride_pan"
+
+/obj/item/clothing/accessory/pridepin/lesbian
+	name = "lesbian pride pin"
+	icon_state = "pride_lesbian"
+
+/obj/item/clothing/accessory/pridepin/intersex
```
Comment (kreeperHLC) [inline code/modules/clothing/under/accessories/pins.dm:34]:
```suggestion
/obj/item/clothing/accessory/pride_pin
	name = "pride pin"
	desc = "A Nanotrasen Diversity & Inclusion Center-sponsored holographic pin to show off your pride, reminding the crew of their unwavering commitment to equity, diversity, and inclusion!"

	icon_state = "pride"
	icon = 'icons/clothing/accessories/pins.dmi'

	icon_override = 'icons/mob/pins.dmi'

/obj/item/clothing/accessory/pride_pin/bi
	name = "bisexual pride pin"
	icon_state = "pride_bi"

/obj/item/clothing/accessory/pride_pin/trans
	name = "transgender pride pin"
	icon_state = "pride_trans"

/obj/item/clothing/accessory/pride_pin/ace
	name = "asexual pride pin"
	icon_state = "pride_ace"

/obj/item/clothing/accessory/pride_pin/enby
	name = "non-binary pride pin"
	icon_state = "pride_enby"

/obj/item/clothing/accessory/pride_pin/pan
	name = "pansexual pride pin"
	icon_state = "pride_pan"

/obj/item/clothing/accessory/pride_pin/lesbian
	name = "lesbian pride pin"
	icon_state = "pride_lesbian"

/obj/item/clothing/accessory/pride_pin/intersex
```

File: code/modules/client/preference_setup/loadout/items/accessories.dm [hunk 2]
```diff
@@ -693,3 +693,21 @@
...
+	. = ..()
+	var/list/selection = list()
+	selection["rainbow pride pin"] = /obj/item/clothing/accessory/pridepin
+	selection["bisexual pride pin"] = /obj/item/clothing/accessory/pridepin/bi
+	selection["transgender pride pin"] = /obj/item/clothing/accessory/pridepin/trans
+	selection["asexual pride pin"] = /obj/item/clothing/accessory/pridepin/ace
+	selection["non-binary pride pin"] = /obj/item/clothing/accessory/pridepin/enby
+	selection["pansexual pride pin"] = /obj/item/clothing/accessory/pridepin/pan
+	selection["lesbian pride pin"] = /obj/item/clothing/accessory/pridepin/lesbian
+	selection["intersex pride pin"] = /obj/item/clothing/accessory/pridepin/intersex
```
Comment (kreeperHLC) [inline code/modules/client/preference_setup/loadout/items/accessories.dm:712]:
```suggestion
	selection["rainbow pride pin"] = /obj/item/clothing/accessory/pride_pin
	selection["bisexual pride pin"] = /obj/item/clothing/accessory/pride_pin/bi
	selection["transgender pride pin"] = /obj/item/clothing/accessory/pride_pin/trans
	selection["asexual pride pin"] = /obj/item/clothing/accessory/pride_pin/ace
	selection["non-binary pride pin"] = /obj/item/clothing/accessory/pride_pin/enby
	selection["pansexual pride pin"] = /obj/item/clothing/accessory/pride_pin/pan
	selection["lesbian pride pin"] = /obj/item/clothing/accessory/pride_pin/lesbian
	selection["intersex pride pin"] = /obj/item/clothing/accessory/pride_pin/intersex
```

File: code/modules/clothing/under/accessories/pins.dm (hunk 1 above)
Comment (intercepti0n) [reply code/modules/clothing/under/accessories/pins.dm:34]: No thanks, Bro!

**Comment by Arrow768** [comment]
Closed as not wanted.

We have no interest in transplanting current day issues into the game.

**Comment by intercepti0n** [comment]
I feel offended.

**Comment by Wildwestmars** [comment]
damn. 
pensive-face

**Pull request closed without merging**